Orange/tests/test_majority.py

```diff
@@ -13,55 +13,51 @@ class MajorityTest(unittest.TestCase):
     @classmethod
     def setUpClass(cls):
         cls.iris = Table('iris')
+        cls.learn = MajorityLearner()
 
     def test_majority(self):
         nrows = 1000
         ncols = 10
         x = np.random.random_integers(1, 3, (nrows, ncols))
         y = np.random.random_integers(1, 3, (nrows, 1)) // 2
         t = Table(x, y)
-        learn = MajorityLearner()
-        clf = learn(t)
+        clf = self.learn(t)
 
         x2 = np.random.random_integers(1, 3, (nrows, ncols))
         y2 = clf(x2)
-        self.assertTrue((y2 == 1).all())
+        self.assertEqual(y2.all(), 1)
 
     def test_weights(self):
         nrows = 100
         ncols = 10
         x = np.random.random_integers(1, 3, (nrows, ncols))
         y = np.array(70*[0] + 30*[1]).reshape((nrows, 1))
-        heavy_class = 1
-        w = (y == heavy_class) * 2 + 1
+        w = (y == 1) * 2 + 1
```

Comment:
`heavy_class` was OK. [Prefer symbols to magic numbers](https://en.wikipedia.org/wiki/Magic_number_%28programming%29#Unnamed_numerical_constants).